index.js

```diff
@@ -78,6 +85,7 @@ export const prefetchConfiguration = async ({
   serviceConfiguration,
   dangerouslyAllowInsecureHttpRequests = false,
   customHeaders,
+  connectionTimeoutSeconds = 0,
 }) => {
```

Comment:
I was unable to pass a default value of null here because of the Java type. Even though Double can hold a `null` value, the conversion by React-Native of this arg to the Java type Double failed for null.